packages/parser/src/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { JSXElement, createMemo } from "solid-js";
+
+export function createParser(id?: string) {
+  const $TOKEN = Symbol(id || "solid-parser");
+
+  function tokenize<
+    TToken extends Meta & {
+      props: unknown;
+    }
+  >(meta: Omit<TToken, "props">): (props: TToken["props"]) => JSXElement {
+    return (props: any) => {
+      return Object.assign(
+        () => {
+          console.info("tokens can only be rendered inside a Parser");
```

Comment:
warnings we usually put behind a dev check so that it's only included in the dev build
```suggestion
          process.env.DEV && console.info("tokens can only be rendered inside a Parser");
```

A `--dev` flag needs to be included in the build script to make it work though
-> `jiti ../../scripts/build.ts --dev`